packages/fungible-token/bridge-fungible-token/tests/functions/bridge/mod.rs

```diff
@@ -458,6 +489,8 @@ mod success {
 mod revert {
     use super::*;
     use crate::utils::setup::get_asset_id;
+    use crate::utils::setup::RefundRegisteredEvent;
```

Comment:
Since these both come from the same place (setup) I would import them on 1 line with `{get_asset_id, RefundRegisteredEvent}`

Reply:
actually deleting it since it is not used